README.adoc

```diff
@@ -69,15 +75,15 @@ mv README.adoc README.meta.adoc
 
 . Edit your project metafile.
 +
-In your favorite text/code editor, open `_data/meta.yml` and fill out the required info, then *save the file*.
+In your favorite text/code editor, open `data/meta.yml` and fill out the required info, then *save the file*.
 
 . Run LiquiDoc with the LDCMF initialization config.
 +
 ----
 bundle exec liquidoc -c _init/init.yml
 ----
 
-. You can now review and edit your `README.adoc` and `src/index.adoc` files with custom content.
+. You can now review and edit your `README.adoc` and `content/index.adoc` files with custom content.
```

Comment:
This resolves to '...edit your files with custom content'. Maybe 'and populate your content/index/adoc files with custom content.'